test/test_utils/test_sklearn_wrapper.py

```diff
@@ -0,0 +1,113 @@
+'''
+Created on July 27, 2017
+
+@author: Jan N. van Rijn
+'''
+import unittest
+
+import numpy as np
+
+from functools import partial
+
+from sklearn import datasets
+from sklearn.dummy import DummyClassifier
+from sklearn.model_selection._split import StratifiedKFold
+from sklearn.tree import DecisionTreeClassifier
+from sklearn.utils.testing import assert_raises
+
+from ConfigSpace.hyperparameters import CategoricalHyperparameter, UniformIntegerHyperparameter
+
+from smac.configspace import ConfigurationSpace
+from smac.scenario.scenario import Scenario
+from smac.facade.smac_facade import SMAC
+from smac.utils.sklearn_wrapper import ModelBasedOptimization
+
+
+class SklearnWrapperTest(unittest.TestCase):
+
+    @staticmethod
+    def _config_space_to_param_grid(config_space):
+        ''' Converts config space into the scikit learn param grid format. Could be useful utility function, but
+        does not contain the right functionality yet.
+        '''
+        param_grid = {}
+
+        for hyperparameter in config_space.get_hyperparameters():
+            if isinstance(hyperparameter, CategoricalHyperparameter):
+                param_grid[hyperparameter.name] = hyperparameter.choices
+            elif isinstance(hyperparameter, UniformIntegerHyperparameter):
+                if hyperparameter.log:
+                    raise ValueError('Parameter logscale unimplemented: %s' + str(hyperparameter.__class__.__name__))
```

Comment:
Agreed, but this function is not a testcase. This is more a utility function, that would not have been misplaced in the main codebase. However, due to the absence of logscale support of this function, I do not find it mature enough for that. 

If something goes wrong in this function, this does not indicate that the code is broken, rather it indicates that the test case is set up wrongly. 

Your decision. Raise or assert?